composer.json

```diff
@@ -21,23 +21,26 @@
         "forum": "https://discourse.laminas.dev"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
     },
     "extra": {
         "laminas": {
             "config-provider": "Mezzio\\Csrf\\ConfigProvider"
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "php": "^7.4 || ~8.0.0 || ~8.1.0",
```

Comment:
Please see: https://github.com/mezzio/mezzio-authorization-acl/pull/16#discussion_r780063669

Reply:
I've adjusted the title - will result in decent release notes.